packages/components/src/button/style.scss

```diff
@@ -256,7 +256,7 @@
 
 	// Fixes a Safari+VoiceOver bug, where the screen reader text is announced not respecting the source order.
 	// See https://core.trac.wordpress.org/ticket/42006 and https://github.com/h5bp/html5-boilerplate/issues/1985
-	.screen-reader-text {
+	.components-visually-hidden {
```

Comment:
You probably need to include both. `style.scss` is shared between the front and the editor.

Reply:
Ignore it, I thought it's code for the Button block 🙃